bin/generateFixtures.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+const faker = require('faker');
+
+const random = max => Math.floor(Math.random() * max) + 1;
+
+const posts = [...Array(15).keys()].map(id => ({
```

Comment:
The more you know..
`Array.from({ length: 15 }, (v, k) => k+1)`

Or much more readable !
`(function(n){for(a=[];n--;a[n]=n+1);return a})(15)`